dali/test/python/test_dali_cpu_only.py

```diff
@@ -891,6 +891,17 @@ def get_data():
         return out
     check_single_input(fn.squeeze, axis_names="YZ", get_data=get_data, input_layout="HWCYZ")
 
+def test_select_cpu():
```

Comment:
I think you need to update test_dali_variable_batch_size.py as well.

Reply:
The regular test uses variable batch size.

Reply:
We need as the test_dali_variable_batch_size.py checks if all operators are tested. There is no other way to enforce this kind of test.